lighthouse-viewer/app/src/psi-api.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * @license Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+/** @typedef {{lighthouseResult?: LH.Result, error?: {message: string}}} PSIResponse */
+
+const PSI_URL = 'https://www.googleapis.com/pagespeedonline/v5/runPagespeed';
+const PSI_KEY = 'AIzaSyAjcDRNN9CX9dCazhqI4lGR7yyQbkd_oYE';
+const PSI_DEFAULT_CATEGORIES = [
+  'performance',
+  'accessibility',
+  'seo',
+  'best-practices',
+  'pwa',
+];
+
+/**
+ * Wrapper around the PSI API for fetching LHR.
+ */
+class PSIApi {
+  /**
+   * @param {PSIParams} params
+   * @return {Promise<PSIResponse>}
+   */
+  fetchPSI(params) {
+    const apiUrl = new URL(PSI_URL);
+    for (const kv of Object.entries(params)) {
```

Comment:
i'd be fine with `for (let [name, value] of ...`

yes the name wouldnt be const but i think its worth it.

Reply:
![image](https://user-images.githubusercontent.com/4071474/63191768-3313d380-c01e-11e9-84b6-bc978c633afc.png)

eslint thinks otherwise. i'll just mute  the lint  error tho